src/index.css

```diff
@@ -1,6 +1,294 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
 body {
   margin: 0;
-  font-family: cursive;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: #f5f6fa;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: var(--white-two);
+}
+
+.heading-nav {
+  display: flex;
+  align-items: center;
+  gap: 3rem;
+}
+
+nav {
+  display: flex;
+  gap: 2rem;
+}
+
+nav a {
+  text-decoration: none;
+  font-family: var(--Montserrat);
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  opacity: 0.5;
+}
+
+form {
+  margin-bottom: 4rem;
+  display: flex;
+  gap: 2rem;
+}
+
+input {
+  width: 40%;
+  padding: 1rem 0.5rem;
+  border-radius: 2px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+  font-size: 1rem;
+  font-family: var(--Montserrat);
+  font-weight: bolder;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: var(--very-light-pink);
+}
+
+button[type='submit'] {
+  width: 10%;
+  background-color: var(--azure);
+  border: none;
+  font-family: var(--RobotoSlab);
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: var(--white-two);
+}
+
+.active {
+  opacity: 1;
+}
+
+.Bookstores {
+  padding-left: 3rem;
+  font-family: var(--Montserrat);
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--azure);
+}
+
+.L-Panel {
+  margin: 2rem auto;
+  width: 85%;
+  padding: 2rem 4rem 1.625rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+}
```

Comment:
- [ ] It would be nice if you could align the book items to be in a straight line from the top of the navbar items just as it's well aligned on the design given. Kindly look at the screenshot for a better guide on the alignment. 👍🏽

**The Original Design**

<img width="871" alt="Screenshot 2023-04-07 at 10 03 31 AM" src="https://user-images.githubusercontent.com/50941074/230591317-0fa1d5c9-ae5b-4e92-9fc9-659734246337.png">

**Your Design**

<img width="1752" alt="Screenshot 2023-04-07 at 10 04 28 AM" src="https://user-images.githubusercontent.com/50941074/230591352-1577943c-6c69-4bb3-97bb-26a36cb2c24e.png">